virttest/nfs.py

```diff
@@ -225,7 +225,7 @@ def setup(self):
         if not os.path.isdir(self.mount_dir):
             os.makedirs(self.mount_dir)
             self.rm_mount_dir = True
-        self.mount()
+        utils_misc.wait_for(self.mount(), 5, text="Waiting for nfs mount")
```

Comment:
actually, I'm not sure if I shall retry the nfs mount or not. The issue is sometimes the mount info just not timely updated in /etc/mtab after nfs mount. So it may cause nfs pool fail to create. In this case, I think wait_for(is_mounted) is more suitable than nfs retry. I changed the code, pls help review again.